frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -962,28 +966,28 @@ end
 function ReaderFooter:textOptionTitles(option)
     local symbol = self.settings.item_prefix
     local option_titles = {
-        all_at_once = _("Show all at once"),
-        reclaim_height = _("Reclaim bar height from bottom margin"),
+        all_at_once = _("Show all selected items at once"),
+        reclaim_height = _("Overlap status bar"),
         bookmark_count = T(_("Bookmark count (%1)"), symbol_prefix[symbol].bookmark_count),
```

Comment:
If we change to "Annotation count" what will be the abbreviation, instead of "BM"?

Reply:
I suppose this is in reference to #11822, and you are asking about user showing letters instead of icons. the title here would then become "Annotation count" (I guess) so... AC?? is that already used?

Reply:
That's for you to decide and fix (c.f., the `symbol_prefix.letters` table at the top of the file) ;).

Reply:
we are not changing that here (or at least I am not going to), that is a task for another PR as there are a lot more instances of "Bookmarks" that would need changing. one step at a time please